tools/linker/MonoTouch.Tuner/ListExportedSymbols.cs

```diff
@@ -64,23 +93,36 @@ protected override void ProcessAssembly (AssemblyDefinition assembly)
 			if (!hasSymbols)
 				return;
 
+			var modified = false;
 			foreach (var type in assembly.MainModule.Types)
-				ProcessType (type);
+				modified |= ProcessType (type);
+
+			if (modified) {
+				var action = Context.Annotations.GetAction (assembly);
+				switch (action) {
+				case AssemblyAction.Copy:
```

Comment:
Are you expecting more cases anytime soon or why choosing a switch for just one case?
You could avoid a nested if by doing `if (modified && Context.Annotations.GetAction (assembly) == AssemblyAction.Copy) { ... }`

Reply:
I simplified this a bit